flows/engine/session.go

```diff
@@ -7,24 +7,36 @@ import (
 	"github.com/nyaruka/goflow/flows"
 	"github.com/nyaruka/goflow/flows/events"
 	"github.com/nyaruka/goflow/flows/runs"
+	"github.com/nyaruka/goflow/flows/waits"
 	"github.com/nyaruka/goflow/utils"
 )
 
+// used to trigger a new flow run in the event loop
+type flowTrigger struct {
```

Comment:
Ya, at first wasn't so hot about it because of the overlap with Trigger in RapidPro but I guess they are really the same thing when it comes down to it. So I'm cool with it.